app/Providers/AppServiceProvider.php

```
@@ -127,4 +125,31 @@ protected function humanizePermission(string $name) : string
         // fallback: just prettify words
         return ucfirst(str_replace('.', ' ', $name));
     }
+
+    /**
+     * Register listener for broadcast events to enable event replay
+     */
+    private function registerBroadcastEventListener(): void
+    {
+        // Listen for all broadcast events and record them
+        try {
+            \Illuminate\Support\Facades\Event::listen('*', function ($eventName, $payload) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$eventName'. (undefined)

(UnusedFormalParameter)